### DISCUSSION TITLE
[Feature Request] Follow link feature for custom uri schemes

### DISCUSSION THREAD

**Discussion by mikulassychra** [microsoft/monaco-editor#4025] [Q&A]
Hello,

I was trying to find solution how to make a clickable (cmd + click / ctrl + click) link from non-http link.

Here is how it works with urls automatically:
![image](https://github.com/microsoft/monaco-editor/assets/126165863/5bba2eb1-aa8e-4fd4-a58b-a2dc60966fa5)

Here is where I would like to have same functionality:
![image](https://github.com/microsoft/monaco-editor/assets/126165863/620c118c-7253-4f77-b36d-c7fdbf70a4a6)

Anyone have a solution, please?

Thanks!

**Comment by ben519** [comment]
I'm also looking for something like this.
